store/src/java/com/zimbra/cs/mailbox/AutoSendDraftTask.java

```diff
@@ -102,6 +115,15 @@ public static void cancelTask(int draftId, int mailboxId) throws ServiceExceptio
      * @param autoSendTime
      * @throws ServiceException
      */
+    public static void scheduleTask(int draftId, int mailboxId, int delegatorMailboxId, long autoSendTime) throws ServiceException {
```

Comment:
@Gopal-Moolchandani this is overloaded method, optimize it with calling this method from method on line #127
`scheduleTask(draftId, mailboxId, 0, autoSendTime);`